src/logging.h

```diff
@@ -68,11 +68,12 @@ namespace BCLog {
         ALL         = ~(uint32_t)0,
     };
     enum class Level {
-        Debug = 0,
-        None = 1,
-        Info = 2,
-        Warning = 3,
-        Error = 4,
+        Trace = 0,
```

Comment:
I think some docs on the different levels wouldn't hurt. For example, it's not quite clear to me how you choose between `Trace` and `Debug`.

Reply:
Generally, the distinction is:
- debug is for reasonably noisy logging, but not high enough volume to make it unusable in a production scenario
- tacing is super high detail, step by step, and will slow down the application and you'd definitely never want to enable it for a category unless you're developing or trouble-shooting a specific issue

Reply:
Thanks! Added documentation to the Level enum class.